code/modules/vehicles/scooter.dm

```diff
@@ -116,24 +129,43 @@
 		return
 
 	var/mob/living/skater = buckled_mobs[1]
-	skater.adjustStaminaLoss(instability*0.5)
-	if (skater.getStaminaLoss() >= 100)
+	skater.adjustStaminaLoss(instability*0.3)
+	if(skater.getStaminaLoss() >= 100)
 		obj_flags = CAN_BE_HIT
 		playsound(src, 'sound/effects/bang.ogg', 20, TRUE)
 		unbuckle_mob(skater)
 		var/atom/throw_target = get_edge_target_turf(src, pick(GLOB.cardinals))
 		skater.throw_at(throw_target, 2, 2)
 		visible_message(span_danger("[skater] loses [skater.p_their()] footing and slams on the ground!"))
-		skater.Paralyze(40)
+		skater.Paralyze(4 SECONDS)
 		grinding = FALSE
 		icon_state = "[initial(icon_state)]"
 		return
 	playsound(src, 'sound/vehicles/skateboard_roll.ogg', 50, TRUE)
-	if(prob(25))
-		var/turf/location = get_turf(src)
-		if(location)
+	var/turf/location = get_turf(src)
+
+	if(location)
+		if(prob(25))
 			location.hotspot_expose(1000,1000)
-		sparks.start() //the most radical way to start plasma fires
+			sparks.start() //the most radical way to start plasma fires
+
+		for(var/mob/living/carbon/victim in location)
+			if(victim.body_position == LYING_DOWN)
+				var/total_brute = victim.getBruteLoss()
+				playsound(location, 'sound/items/trayhit2.ogg', 40)
+				if(victim.stat == DEAD && total_brute >= 175)
+					victim.gib() // Beaten and brutalised, your body gives up
```

Comment:
This is a definite no from me.

Reply:
I think I understand where you are coming from, but could you elaborate?

Reply:
I'd rather not expand on the ways of round removal on a feature like this.
The less balance concerns you introduce, the better.

Reply:
Yeah, gibbing is super over the top here in this scenario mostly because it doesn't make sense and I'll remove it.

Generally, I'm all for more round removal and wacky balance but I suppose that's more of a philosophical issue rather than a coding one huh. Balance concerns are the fun part after all.